events/flavorGNU.txt

```diff
@@ -188,11 +188,16 @@ country_event = {
 		name = flavor_gnu.6.a
 		every_owned_province = {
 			limit = {
-				OR = {
-					area = Dragon_islands
-					area = Eastern_dragon_islands
-					area = Western_dragon_islands
+				orbital_region = {
+				limit = {
+					NOT = { is_core = ROOT }
+					NOT = { is_permanent_claim = ROOT }
+					NOT = { area = crack_of_spacetime }
+					NOT = { area = okazaki_state }
+					NOT = { area = yumemi_space_state }
 				}
+				add_permanent_claim = ROOT
+			}
 			}
 			add_province_modifier = {
 				name = gnu_heaven_to_living_hell
```

Comment:
An effect is trying to be applied in a limit block...? is somewhat confusing, not sure intent.